core/hash.go

```diff
@@ -0,0 +1,37 @@
+package core
+
+import (
+	lib "github.com/NethermindEth/crypto-cpp"
+	"github.com/NethermindEth/juno/core/felt"
+)
+
+// Implements the [Pedersen hash]
```

Comment:
I think the comments on the two Pedersen functions are swapped. Please feel free to ignore and resolve these comments if I'm missing something.

```suggestion
// PedersenDigest implements [Pedersen array hashing].
```